### ISSUE TITLE
Training with only CelebHQ dataset

### ISSUE THREAD

**Issue by Qiulin-W** [mindslab-ai/faceshifter#13]
Thanks for the great work! 

When I try to train the AEI-Net with 30k images from celebHQ dataset using 6 P40 32G GPUs, I got the training curve as below:
![Screenshot from 2020-11-23 11-40-54](https://user-images.githubusercontent.com/37483535/99927851-cc817780-2d81-11eb-934c-16856195517c.png)
![Screenshot from 2020-11-23 11-41-05](https://user-images.githubusercontent.com/37483535/99927862-d30fef00-2d81-11eb-83c7-65edf024bf27.png)
![Screenshot from 2020-11-23 11-41-17](https://user-images.githubusercontent.com/37483535/99927865-d60adf80-2d81-11eb-8f20-b52dafd13785.png)
![Screenshot from 2020-11-23 11-41-25](https://user-images.githubusercontent.com/37483535/99927870-d86d3980-2d81-11eb-857b-fd90c229583c.png)
![Screenshot from 2020-11-23 11-41-34](https://user-images.githubusercontent.com/37483535/99927873-dacf9380-2d81-11eb-8577-fdfae44ffde4.png)
![Screenshot from 2020-11-23 11-41-41](https://user-images.githubusercontent.com/37483535/99927877-e0c57480-2d81-11eb-9be8-a8f6f7c15ba8.png)

All the other setting are set by default and the generated swap faces are also weird:
![00001](https://user-images.githubusercontent.com/37483535/99927942-1b2f1180-2d82-11eb-95f9-466bd0b5a727.jpg)
![00002](https://user-images.githubusercontent.com/37483535/99927944-1c603e80-2d82-11eb-8c3d-3b38f2f7ca60.jpg)
![output2](https://user-images.githubusercontent.com/37483535/99927946-1e2a0200-2d82-11eb-9b9e-49c7dd7cd372.png)

Should I continue training or any sugguestions? Thanks in advance!

**Comment by usingcolor** [comment]
Hi! Thanks for your compliments.
Training with only CelebA-HQ is quite a dangerous choice.
The dataset bias can be affected to total loss.
In your loss graph, reconstruction loss is very high. 
Maybe you should try to increase the coefficient of reconstruction loss if you wanna train with only CelebA-HQ.

Be careful about overfitting.

**Comment by Qiulin-W** [comment]
> Hi! Thanks for your compliments.
> Training with only CelebA-HQ is quite a dangerous choice.
> The dataset bias can be affected to total loss.
> In your loss graph, reconstruction loss is very high.
> Maybe you should try to increase the coefficient of reconstruction loss if you wanna train with only CelebA-HQ.
> 
> Be careful about overfitting.

Thank you so much for your reply!

By the way, what is the expected value of reconstruction loss of your well-trained model? If possible, could you share your loss graph as a reference? Did you set grad_clip to zero?

**Comment by usingcolor** [comment]
1.  reconstruction loss should be around 1e-3~1e-4 for a well-trained model. You can see the reconstructed example in my colab example which is presently added.
2. No. I can't share the loss graph.
3. Yes. I didn't clip the gradient. The grad_clip option to zero means no clip.

**Comment by coranholmes** [comment]
Could you please tell me how long does it take to finish the training?

**Comment by Qiulin-W** [comment]
> Could you please tell me how long does it take to finish the training?

For celebHQ dataset only, it takes 2-3 hours for an epoch.

**Comment by hanikh** [comment]
> > Could you please tell me how long does it take to finish the training?
> 
> For celebHQ dataset only, it takes 2-3 hours for an epoch.

can you benefit from multi-GPU training? does it really increase the training speed?

**Comment by rainq22** [comment]
已经过去四年了，请问下你能分享下arcface.pth文件吗，链接已经失效了，感激不尽
